cmd/kubelet/app/options/options.go

```diff
@@ -265,6 +269,7 @@ func applyLegacyDefaults(kc *kubeletconfig.KubeletConfiguration) {
 type KubeletServer struct {
 	KubeletFlags
 	kubeletconfig.KubeletConfiguration
+	kubeletconfig.KubeletInstanceConfiguration
```

Comment:
I think giving this a field name will fix the go vet error. Looks like a name collision on the json tags when embedding directly, since TypeMetas conflict between KubeletConfiguration and KubeletInstanceConfiguration (I'd expect the same error for any duplicate fields between the two types, so we'd ultimately have to do this anyway). Something like `instanceConfig` should be ok.